server/api-docs.apib

```diff
@@ -73,6 +73,41 @@ Creates Card
         ]
         ```
 
+## Card [/api/v1/cards/{id}]
+
++ Parameters
+    + id: 1 (required)
+
+### DELETE
+
++ Request (application/json)
+
++ Response 204
+
+## Card not found [/api/v1/cards/{id}]
+
++ Parameters
+    + id: 1000 (required)
+
+### DELETE
+
++ Request (application/json)
+
++ Response 400
+
+  - Attributes (object)
+      + error (string) - Error type
+      + message (string) - Error message
+
+  - Body
+
+      ```
+      {
+        error: "Bad Request",
+        message: "Card not found"
+      }
+      ```
```

Comment:
Yeah, apiblueprint is a little bit unclear base we can't separate blocks like we would like to. Super strict rules for that. Here is overwritten docs for provided above (I've rewritten this part with a look on [docs](https://github.com/apiaryio/api-blueprint/edit/master/examples/Real%20World%20API.md)):

```
## Card [/api/v1/cards/{id}]

- Parameters
    + id: 1 (required)

### DELETE

- Request (application/json)
- Response 204

- Request (application/json)

    When record not found

    - Parameters
        + id: 1000 (required)

- Response 400

    - Attributes (object)
        + error (string) - Error type
        + message (string) - Error message

    - Body

        ```
        {
          error: "Bad Request",
          message: "Card not found"
        }
        ```
```

Reply:
@AleksSenkou 
it fails everytime:
```

- Request (application/json)
    - Parameters
        + id: 1000 (required)

- Response 400

    - Attributes (object)
        + error (string) - Error type
        + message (string) - Error message

    - Body

        ```
        {
          error: "Bad Request",
          message: "Card not found"
        }
        ```

```

Reply:
Fixed during PP

Reply:
`(application/json; charset=utf-8)`